apps/demo-app/app/components/KPICard.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import { iconExpandWindow } from "@sit-onyx/icons";
+import { OnyxCard, OnyxIcon, OnyxTag, type OnyxColor } from "sit-onyx";
+
+const props = defineProps<{
+  headline: string;
+  value: number;
+  color: OnyxColor;
+  percent: number;
+  icon: string;
+}>();
+</script>
+
+<template>
+  <OnyxCard class="card" clickable>
+    <div class="card__header">
+      {{ props.headline }}
+      <OnyxIcon :icon="iconExpandWindow" />
+    </div>
+
+    <div class="card__content">
+      <span class="card__value"> {{ props.value }} </span>
+      <OnyxTag
+        :label="props.percent + ' %'"
+        :icon="props.icon"
+        density="compact"
+        :color="props.color"
+      />
+    </div>
+  </OnyxCard>
+</template>
+
+<style lang="scss" scoped>
+.card {
+  width: 16rem;
```

Comment:
Since we are using a grid layout we should remove the custom width so they adapt correctly